server/sonar-db-dao/src/test/java/org/sonar/db/rule/RuleDbTester.java

```diff
@@ -111,12 +118,43 @@ public RuleDto insertRule() {
     });
   }
 
-  public RuleDto insertRule(OrganizationDto organization, Consumer<RuleDto>... populaters) {
+  @SafeVarargs
+  public final RuleDto insertRule(OrganizationDto organization, Consumer<RuleDto>... populaters) {
     RuleDto ruleDto = newRuleDto(organization);
     Arrays.asList(populaters).forEach(populater -> populater.accept(ruleDto));
     return insertRule(ruleDto);
   }
 
+  @SafeVarargs
+  public final RuleDefinitionDto insertRuleDefinition(Consumer<RuleDefinitionDto>... populaters) {
```

Comment:
I removed that method, but kept the varargs. It allows to reuse predefined consumers, which I often use for to allow this syntax:
`insert(setTags("a", "b"), setSystemTags("s", "t"));
`
instead of:
`insert(r -> r.setTags(asList("a", "b")).setSystemTags(asList("s", "t")));
`